system/jlib/jlog.cpp

```diff
@@ -2511,17 +2588,18 @@ MODULE_EXIT()
         theReporters[compo] = NULL;
     }
     delete theManager;
```

Comment:
theManager->Release()?  I don't think the destructor is virtual.

Reply:
will change